dnf-docker-test/features/steps/repo_steps.py

```diff
@@ -244,6 +246,10 @@ def given_repository_with_packages(ctx, rtype, repository):
     ctx.table.add_row(["",         "enabled",  "False"])
     ctx.table.add_row(["",         "gpgcheck", "False"])
     ctx.table.add_row(["",         "baseurl",  "{!s}://{!s}".format(rtype, repopath)])
+    if rtype == 'https':
+        ctx.table.add_row(["",         "sslcacert",  "/etc/pki/tls/certs/testcerts/ca/cert.pem"])
```

Comment:
don't bother with additional alignments ;)

I just wanted all parameters to be on the same level.